jadx-gui/src/main/java/jadx/gui/ui/codearea/JNodeAction.java

```diff
@@ -31,21 +30,12 @@ public boolean isActionEnabled(@Nullable JNode node) {
 		return node != null;
 	}
 
-	public void addKeyBinding(KeyStroke key, String id) {
-		UiUtils.addKeyBinding(codeArea, key, id, new AbstractAction() {
-			@Override
-			public void actionPerformed(ActionEvent e) {
-				node = codeArea.getNodeUnderCaret();
-				if (isActionEnabled(node)) {
-					runAction(node);
-				}
-			}
-		});
-	}
-
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		runAction(node);
+		node = codeArea.getNodeUnderCaret();
+		if (isActionEnabled(node)) {
+			runAction(node);
+		}
```

Comment:
```suggestion
		if (JadxGuiAction.COMMAND.equals(e.getActionCommand())) {
			node = codeArea.getNodeUnderCaret();
			if (isActionEnabled(node)) {
				runAction(node);
			}
		} else {
			runAction(node);
		}
```

Reply:
We don't need to use getNodeUnderCaret if action was triggered by mouse (i.e popup menu)